docs/community/news.md

```diff
@@ -5,33 +5,37 @@ hide:
 ---
 
 # News and Announcements
-* **March 2023:** New Stack blog article by Michael Kalantar. [Iter8: Simple A/B/n Testing of Kubernetes Apps, ML Models](https://thenewstack.io/iter8-simple-a-b-n-testing-of-kubernetes-apps-ml-models/)
 
-* **February 2023:** DZone article by Alan Cha. [Automated Performance Testing With ArgoCD and Iter8](https://dzone.com/articles/automated-performance-testing-with-argocd-and-iter8)
+<!-- TODO: Many of these are incompatible now. Do we want to make a break like this? -->
+???+ note "Iter8 v0.15 and older"
```

Comment:
We have a separation for v0.7 below. I thought that this may be because it was an incompatible version. v0.16 breaks things as well so should these previous articles also be separated?